<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>622</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Noteeds</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,1">
    <item row="0" column="0">
     <widget class="QLabel" name="repositoriesLabel">
      <property name="text">
       <string>Repositories:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;Search:</string>
         </property>
         <property name="buddy">
          <cstring>textInput</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textInput"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="resultsTree">
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="TextView" name="textView"/>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="settingsAction"/>
    <addaction name="editAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="hideAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="viewMenu"/>
  </widget>
  <widget class="QDockWidget" name="dock">
   <property name="windowTitle">
    <string>Log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeView" name="logTable">
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="exitAction">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="hideAction">
   <property name="text">
    <string>&amp;Hide</string>
   </property>
   <property name="toolTip">
    <string>Hide window to systray</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="settingsAction">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="testAction">
   <property name="text">
    <string>&amp;Test</string>
   </property>
  </action>
  <action name="editAction">
   <property name="text">
    <string>&amp;Edit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextView</class>
   <extends>QPlainTextEdit</extends>
   <header>noteeds.gui.text_view</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>resultsTree</tabstop>
  <tabstop>textView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
